pylint/lint/pylinter.py

```diff
@@ -1046,7 +1057,10 @@ def _iterate_file_descrs(self, files_or_modules):
     def _expand_files(self, modules):
         """get modules and errors from a list of modules and handle errors"""
         result, errors = expand_modules(
-            modules, self.config.black_list, self.config.black_list_re
+            modules,
+            self.config.black_list,
+            self.config.black_list_re,
+            self.config.black_list_paths_re,
```

Comment:
I don't know if we can change from black to ignore everywhere here, if it's not trivial then we should onlny do it for new argument.
```suggestion
            self.config.ignore_list_paths_re",
```

Reply:
I can go with grep over the source and see where black (-list) is still used and take a look if it is trivial to change.

Reply:
(Care need to be taken to not introduce a breaking change that's probably why it was not changed before.)

Reply:
Yeah I'm unsure here, too. I thought the `dest` is really only used internally, but I can change it back to be sure.

Reply:
I partially reverted the changed, except for the new argument.